CONTRIBUTING.md

```diff
@@ -47,3 +48,53 @@ Improve contributing docs and consolidate them in the standard location https://
 ```
 
 ### For more information on Cumulus governance, see the [Cumulus Code Contribution Guidelines](https://docs.google.com/document/d/14J_DS6nyQ32BpeVjdR-YKfzHAzFB299tKghPGshXUTU/edit) and [the Cumulus Wiki](https://wiki.earthdata.nasa.gov/display/CUMULUS/Cumulus).
+
+## Getting started for beginners 
+
+### Pre-requisites
+- [Docker](https://www.docker.com/) a Docker daemon must be running on your local machine. Running Docker Desktop is a simple way to achieve this
+- Node Version Manager ([NVM](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating))
+- Node Package Manager ([NPM](https://github.com/npm/cli))
+
+### Instructions
+1. If you have not done so already, [fork this repo](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo). 
+2. [Clone your forked repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to your local environment.
+```bash
+git clone https://github.com/<your github account name>/cumulus-dashboard.git
+cd cumulus-dashboard
+```
+3. [Create a branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) to develop your code in
+4. Install the correct version fo Node.js for the Cumulus Dashboard and set the version for this project
+```bash
+nvm install v20.12.2
+```
+```bash
+nvm use
+```
+5. Install the cumulus dashboard
+```bash
+npm install
+```
+6. Run the unit tests for cumulus dashboard. Do not proceed to the subsequent steps until all these tests have passed.
+```bash
+npm test
```

Comment:
This command is for both eslint and unit tests. If you want them to run the unit tests the command is `npm test-unit-only`.